R/fit_classifier.R

```diff
@@ -1 +1,13 @@
-fit_classifier <- function(){}
+#' Fit dummy classifier and Logistic regression models 
+#'
+#' @return A data frame
+#' @export
+#'
+#' @examples
+#' fit_classifier(data, target_col = 'income', categorical_features='occupation')
+#' fit_classifier(data, target_col = 'income', numeric_feats = ['age', 'fnlwgt', 'hours.per.week', 'education.num', 'capital.gain', 'capital.loss'], categorical_feats=['genre'], cv=10)
```

Comment:
in roxygen, numeric_feats should be list('age', 'fnlwgt')..... the syntax you have used is for python